### DISCUSSION TITLE
Wait what if I use both rpm and moar

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#43] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30536275/158941543-2e38730a-ca5c-43eb-b683-84b5b156b103.jpeg)

**Comment by FuryFire2004** [comment]
I already use both together as it is

**Comment by zer0Kerbal** [comment, marked as answer]
All good. The mod is set up for RPM, but it shouldn't be all that difficult to make a RPM->MAS patch (when I have time) another one of my mods has it.
